avro-builder/tests/tests-allavro/src/test/java/com/linkedin/avroutil1/builder/DefaultRecordTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2022 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.builder;
+
+import com.linkedin.avroutil1.compatibility.AvroCodecUtil;
+import com.linkedin.avroutil1.compatibility.RandomRecordGenerator;
+import com.linkedin.avroutil1.compatibility.RecordGenerationConfig;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class DefaultRecordTest {
+
+  @Test
+  public void testRoundTripSerializationArrayOfStrings() throws Exception {
```

Comment:
nit - method name does not accurately describe whats being tested (no arrays of strings)